mindsdb/integrations/handlers/unify_handler/requirements.txt

```diff
@@ -0,0 +1 @@
+unifyai==0.9.2
```

Comment:
The CI is failing when importing this package because its name and how it is imported is different. Let's add the mapping [here](https://github.com/mindsdb/mindsdb/blob/c1137cf794d9e184771c3c1bad46f87d0e882d1f/tests/scripts/check_requirements.py#L73) to avoid this.